src/resources/views/corporation/ledger/reprocessingfeebymonth.blade.php

```diff
@@ -0,0 +1,70 @@
+@extends('web::corporation.ledger.layouts.view', ['sub_viewname' => 'bountyprizesbymonth'])
+
+@section('title', trans_choice('web::seat.corporation', 1) . ' ' . trans_choice('web::seat.bountyprizesbymonth', 2))
+@section('page_header', trans_choice('web::seat.corporation', 1) . ' ' . trans_choice('web::seat.bountyprizesbymonth', 2))
+
+@section('ledger_content')
+
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h3 class="panel-title">Available Ledgers</h3>
+    </div>
+    <div class="panel-body">
+
+      @foreach ($bountyprizes->chunk(3) as $chunk)
+        <div class="row">
+
+          @foreach ($chunk as $prize)
+            <div class="col-xs-4">
+              <span class="text-bold">
+                <a href="{{ route('corporation.view.ledger.bountyprizesbymonth', ['corporation_id' => $corporation_id, 'year' => $prize->year, 'month' => $prize->month]) }}">
```

Comment:
This route will redirect to bounty prizes from the specified month and year for the current corporation. Is it what you want ?

Your new routes must appear into this file https://github.com/eveseat/web/blob/master/src/Http/Routes/Corporation/View.php#L159-L162. You can follow the pattern but use unique name (`as`), unique route pattern (the string into `get` method) and ensure the `uses` is referring to the proper method from your updated controller.